src/Orleans.Core/Configuration/OptionLogger/OptionFormatterExtensionMethods.cs

```diff
@@ -27,6 +27,15 @@ public static class OptionConfigureExtensionMethods
             return services;
         }
 
+        /// <summary>
+        /// configure option formatter for <typeparam name="TOptions"/>
+        /// </summary>
+        public static IServiceCollection ConfigureFormatter<TOptions>(this IServiceCollection services)
+            where TOptions : class, new()
+        {
+            return services.ConfigureFormatter<TOptions, GenericOptionsFormatter<TOptions>>();
```

Comment:
If this is our default, I don't think we need ass it to these extension functions, only to our default services.

services.AddSingleton(typeof(IOptionFormatter<>), typeof(GenericOptionsFormatter<>));
services.AddSingleton(typeof(IOptionFormatterResolver<>), typeof(GenericOptionsFormatterResolver<>));

This allows any IOptionsFormater mapped to an IOptionsFormatter<O> to be automatically resolved to the generic formatter unless an impl is specifically provided.